pkg/k8sutil/k8sutil.go

```diff
@@ -157,3 +160,18 @@ func CreateOrUpdateEndpoints(eclient clientv1.EndpointsInterface, eps *v1.Endpoi
 
 	return nil
 }
+
+// GetMinorVersion returns the minor version as an integer
+func GetMinorVersion(dclient discovery.DiscoveryInterface) (int, error) {
+	v, err := dclient.ServerVersion()
+	if err != nil {
+		return 0, nil
```

Comment:
True, I don't remember why I ignored it before. Fixed it. And we crash if we fail to get the `MinorVersion` now.

Also, hashicorp's semver is the only one that can parse but it failed on `1.7.0-alpha.2` on minikube (the version returned was garbage anyways like: 0.0.0-master-$Format$ or something). Should we be strict about that too, because compiling k8s from source might produce garbage